tower-scripts/bin/aos-cd-cluster-smoke-test.sh

```diff
@@ -63,24 +63,24 @@ fi
 
 # Using 'EOF' allows you to avoid escaping all uses of $
 ossh "root@${MASTER}" -c "/bin/bash" << 'EOF'
-    set -e
-    oc delete --ignore-not-found project aos-cd-smoke-test-proj
-    sleep 60 # Allow project to be deleted
-    oc new-project aos-cd-smoke-test-proj
+    set -ex
+    PROJ_RANDOM=aos-cd-smoketest-$(shuf -i 100000-999999 -n 1)
+    oc new-project ${PROJ_RANDOM}
     sleep 10  # Allow project to initialize
```

Comment:
This should try `oc get projects` until we see the project show up instead of a blind sleep

Reply:
Agreed.